app/services/pwned_passwords/lookup_password.rb

```diff
@@ -1,6 +1,6 @@
 module PwnedPasswords
   class LookupPassword
-    PWNED_PASSWORD_FILE = Rails.root.join(AppConfig.env.pwned_passwords_file_path).freeze
+    PWNED_PASSWORD_FILE = Rails.root.join(IdentityConfig.store.pwned_passwords_file_path).freeze
```

Comment:
Let's just remove these as configs? Since we hardcode the paths in `deploy/activate.rb` anyways? Or maybe just make this a plain string const and have `deploy/activate` reference this constant? or something?

https://github.com/18F/identity-idp/blob/31cb2200ce96a70d139c241cac4a15be43223660/lib/deploy/activate.rb#L121-L127

Reply:
That's actually the way they were wired up before. We made them configurable so that in a post-chef world we could have these live wherever we need on the disk and change the path the IDP uses with the configs.

Ref: https://github.com/18F/identity-idp/commit/e2b08113467129a24643689db257cabd46062e21

Reply:
PR for that change is here: https://github.com/18F/identity-idp/pull/4797

Reply:
hmmmm I just checked `main` branch and those paths are still hardcoded, and `local_path` is still harcoded and does not referenece `IdentityConfig` at all?

Reply:
Ah, are you referring to w/i the activate script? The idea was for the activate script to keep putting them in the same place. We'll stop running `bin/activate` once we get these files baked in and all of that will go away. At that point we'll use the configs to point to the new paths which may not be under `Rails.root`